os-server/src/main/java/app/openschool/usermanagement/api/dto/UserRegistrationDto.java

```diff
@@ -0,0 +1,50 @@
+package app.openschool.usermanagement.api.dto;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+
+/** Useful Javadoc. */
+public class UserRegistrationDto {
+
+  private static final String PASSWORD_PATTERN =
+      "((?=.*\\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[~!@#$%^&*()_{}]).{8,20})";
+  private static final String PASSWORD_ERROR_MASSAGE =
+      "Password should have 8-20 characters,"
+          + "including upper and lower case letters,"
+          + "numbers and signs(~!@#$%^&*()_{})";
+  private static final String FIRST_NAME_ERROR_MASSAGE = "First name is mandatory";
+  public static final String EMAIL_ERROR_MASSAGE = "Email is mandatory";
+
+  @NotBlank(message = FIRST_NAME_ERROR_MASSAGE)
```

Comment:
don't like this validation but let's try.
We need to keep in mind that all messages should be translated.